### PR TITLE
[New SIG Proposal] add Deepin en español

### PR DESCRIPTION
**Pull request by alvarosamudio**
*(edited by maintainer)*

## Reason

Expand the ecology of the new root community and provide more common apps and gadgets in the form of an app store to provide a better experience for users

## Create a repository

This SIG needs a repository under the deepin-community organization:

- sig-deepin-in-spanish

### CODE REVIEW

File: sig/deepin-en-español/README.md [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+The charter of the SIG. The Charter content can be in somewhere else, if that's the case, you only need to add a link of the Charter to here.
+
+## Discussion channels
+
+Link the public discussion channel(s) for this SIG, and discribe how to join if needed.
+
+## Related Links
+
+- [SIG Repo on GitHub](https://github.com/deepin-community/sig-xxxx)
+- [Other links](url)
```
Comment (BLumia) [inline sig/deepin-en-español/README.md:31]: You need to fill the template with the content about what does this SIG do.

**Review by BLumia — CHANGES_REQUESTED** [review]
Hi!

Thanks for the interest, FYI the Special Interest Group in deepin-community must have a public and clear goal to achieve.  What does this SIG plans to do? Where does the discussion happens , etc. So you really need to fill the template to let other community contributors know about the SIG you'd like to create.

If you don't plan to achieve some kinds of goal but just want to create a end-user forum, then a SIG is not necessary to do so :)

If you still have any question, please feel free to let me know. Thanks!

File: sig/deepin-en-español/README.md (hunk 1 above)
Comment (alvarosamudio) [reply sig/deepin-en-español/README.md:31]: ok

**Review by BLumia — COMMENTED** [review]
Thanks for the update! Once the SIG get created, an associated git repo (`sig-deepin-in-spanish`) will be created which can be used to put the SIG work/goal showcase posts, and the Administrators will be invited to this organization so you guys can maintain that repo.

I might edit the title and description of this PR to ensure it fit the current SIG creation guideline, and I'll approve and merge this PR once I'm ready (probably later today) :)

File: sig/deepin-in-spanish/README.md [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+
+The Deepin en Español community strives to collect the software packages we all need, gather and create quality widgets, take the initiative to adapt them to applications using our app store to bring them together for everyone to access.
+
+What it takes to build an ecosystem is not one party working alone, but everyone acting to contribute to the software ecosystem. We have created an app store to help users get Linux apps easily.
+
+Considering the review environment of community volunteers ready to collaborate, currently our store only works for Debian packages, but in the future we plan to adapt it for linglong packages.
+
+## Discussion channels
+
+-[Telegram Group] (https://t.me/deepinenespanol)
```
Comment (BLumia) [inline sig/deepin-in-spanish/README.md:33]:
```suggestion
- [Telegram Group](https://t.me/deepinenespanol)
```

**Comment by BLumia** [comment]
Seems I cannot apply the change directly so you need to fix that by yourself. After that minor issue gets fixed, I'll approve and get this PR merged.

By the way, the related SIG team has already been created and I already invited the four admins to the team, so they can check their inbox and see if the invitation email is received.

Since SIG requires to have at least one post about the goals that the SIG had achieved quarterly, a SIG repo (https://github.com/deepin-community/sig-deepin-in-spanish) ([site](https://deepin-community.github.io/sig-deepin-in-spanish/)) was created for this purpose so you can put the related posts here. We haven't prepared an English version README file to get you started but it's just a regular [HUGO](https://gohugo.io/) site, so you can simply read the HUGO documentation to learn about how to use it. You can change the content in that repo freely as long as you needed, just make sure there is an RSS atom so we can subscribe :)

**Review by BLumia — APPROVED** [review]

**Pull request merged**